core/templates/dev/head/galleries/Gallery.js

```diff
@@ -196,14 +219,22 @@ oppia.controller('Gallery', [
       }
     });
 
-    // SEARCH FUNCTIONALITY
+    // Transforms the category name of each exploration into a translation id
+    var addTranslationIds = function(allExplorationsInOrder) {
+      for (var i = 0; i < allExplorationsInOrder.length; i++) {
+        allExplorationsInOrder[i].i18nCategory = getI18nId('categories',
```

Comment:
Done, but with the last refactor of the gallery this is not used any more. Do I leave it in case someone wants to show the translated category? Or is it better to remove the dead code?